README.md

```diff
@@ -702,11 +705,80 @@ Name        | Value example | Description
+
+- lock_timeout_seconds - This field is used to specify the time that a device must wait before returning to the secured mode when receiving timed operation modes. Values in range 0...59 must indicate the actual number of seconds. This field must be in range 0...59 if the operation_type field is set to 'timed'. This field might be ommited when operation_type field is set to 'constant'.
+
+- auto_relock_time - This field is used to specify the time setting in seconds for auto-relock functionality. The value 0 must indicate that the auto-relock functionality is disabled and the door lock must not return to secured mode automatically (unless using Timed Operation). Values in range 1...65535 must indicate that the auto-relock functionality is enabled and a device must return to secured mode after the time in seconds indicated by this field. This field will be ignored (so also can be ommited by client) if a device does not support a capability of auto-relock (service property 'supports_auto_relock').
+
+- hold_and_release_time - This field is used to specify the time setting in seconds for letting the latch retracted after a device mode has been changed to unsecured. The value 0 must indicate that the hold and release functionality is disabled and a device must not keep the latch retracted when the door lock mode becomes unsecured. Values in range 1...65535 must indicate that the hold and release functionality is enabled and the door lock latch must keep open according to the time in seconds indicated by this field. This field will be ignored (so also can be ommited by client) if a device does not support a capability of hold and release (service property 'supports_hold_and_release').
+
+- block_to_block - The value true must indicate that the block-to-block functionality is enabled and a device must activate its motors until blocked to try to reach the mode indicated by a cmd.lock.set, even if it detects to be already in a specified mode. The value false must indicate that the block-to-block functionality is disabled and a device may ignore cmd.lock.set message if it detects to be already in the specified mode. This field will be ignored (so also can be ommited by client) if a device does not support a capability of block-to-block (service property 'supports_block_to_block').
+
+- twist_assist - The value true must indicate that the twist assist functionality is enabled and value false indicates that it's disabled. This field will be ignored (so also can be ommited by client) if a device does not support a capability of twist assist (service property 'supports_twist_assist').
+
+Explanation of a possible values for sup_components service property for Z-Wave protocol.
```

Comment:
missing explanation?

Reply:
It's more like, badly written sentence. I didn't want to make an explanation. Just wanted to point out possible values, will fix it :)